From 169bb520f7032ea699d3cdf46121d2819f562b51 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 6 Jun 2023 09:29:51 +0200
Subject: [PATCH] units: change TimeoutSec=0 to TimeoutSec=infinity

Follow-up for #27936

Let's also update a bunch of static unit files, matching what we just
did for the generators.

(cherry picked from commit 1775872679f1ace1771a14294306aa6782b5b263)

Related: #2190226
---
 test/units/autorelabel.service       | 2 +-
 units/rc-local.service.in            | 2 +-
 units/systemd-fsck-root.service.in   | 2 +-
 units/systemd-fsck@.service.in       | 2 +-
 units/systemd-growfs-root.service.in | 2 +-
 units/systemd-growfs@.service.in     | 2 +-
 units/systemd-quotacheck.service.in  | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/test/units/autorelabel.service b/test/units/autorelabel.service
index 1da1002cde..224981f76a 100644
--- a/test/units/autorelabel.service
+++ b/test/units/autorelabel.service
@@ -12,7 +12,7 @@ ConditionPathExists=|/.autorelabel
 [Service]
 ExecStart=sh -x -c 'echo 0 >/sys/fs/selinux/enforce && fixfiles -f -F relabel && rm /.autorelabel && systemctl --force reboot'
 Type=oneshot
-TimeoutSec=0
+TimeoutSec=infinity
 RemainAfterExit=yes
 
 [Install]
diff --git a/units/rc-local.service.in b/units/rc-local.service.in
index 0eee722154..40ffc15c99 100644
--- a/units/rc-local.service.in
+++ b/units/rc-local.service.in
@@ -19,6 +19,6 @@ Wants=network-online.target
 [Service]
 Type=forking
 ExecStart={{RC_LOCAL_PATH}} start
-TimeoutSec=0
+TimeoutSec=infinity
 RemainAfterExit=yes
 GuessMainPID=no
diff --git a/units/systemd-fsck-root.service.in b/units/systemd-fsck-root.service.in
index 8378df84c7..8cfbe7ce98 100644
--- a/units/systemd-fsck-root.service.in
+++ b/units/systemd-fsck-root.service.in
@@ -21,4 +21,4 @@ OnFailureJobMode=replace-irreversibly
 Type=oneshot
 RemainAfterExit=yes
 ExecStart={{ROOTLIBEXECDIR}}/systemd-fsck
-TimeoutSec=0
+TimeoutSec=infinity
diff --git a/units/systemd-fsck@.service.in b/units/systemd-fsck@.service.in
index 06b91aea39..d773229812 100644
--- a/units/systemd-fsck@.service.in
+++ b/units/systemd-fsck@.service.in
@@ -20,4 +20,4 @@ Before=systemd-quotacheck.service shutdown.target
 Type=oneshot
 RemainAfterExit=yes
 ExecStart={{ROOTLIBEXECDIR}}/systemd-fsck %f
-TimeoutSec=0
+TimeoutSec=infinity
diff --git a/units/systemd-growfs-root.service.in b/units/systemd-growfs-root.service.in
index 295bafd5af..1bbd26e0d7 100644
--- a/units/systemd-growfs-root.service.in
+++ b/units/systemd-growfs-root.service.in
@@ -19,4 +19,4 @@ Before=shutdown.target
 Type=oneshot
 RemainAfterExit=yes
 ExecStart={{ROOTLIBEXECDIR}}/systemd-growfs /
-TimeoutSec=0
+TimeoutSec=infinity
diff --git a/units/systemd-growfs@.service.in b/units/systemd-growfs@.service.in
index 7154e4ca76..ff10a3f54e 100644
--- a/units/systemd-growfs@.service.in
+++ b/units/systemd-growfs@.service.in
@@ -20,4 +20,4 @@ Before=shutdown.target
 Type=oneshot
 RemainAfterExit=yes
 ExecStart={{ROOTLIBEXECDIR}}/systemd-growfs %f
-TimeoutSec=0
+TimeoutSec=infinity
diff --git a/units/systemd-quotacheck.service.in b/units/systemd-quotacheck.service.in
index c3e936d220..8e9e9d0691 100644
--- a/units/systemd-quotacheck.service.in
+++ b/units/systemd-quotacheck.service.in
@@ -19,4 +19,4 @@ ConditionPathExists={{QUOTACHECK}}
 Type=oneshot
 RemainAfterExit=yes
 ExecStart={{ROOTLIBEXECDIR}}/systemd-quotacheck
-TimeoutSec=0
+TimeoutSec=infinity
